### PR TITLE
Add an action to rebuild pipeline toolchains and docker images

### PR DESCRIPTION
**Pull request by gabrielBusta**
We can add an action to rebuild the toolchains. Tangentially related to #691

### CODE REVIEW

File: taskcluster/translations_taskgraph/actions/rebuild_toolchains.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    symbol="rebuild-toolchains",
+    description="Create toolchain tasks to rebuild their artifacts.",
+    order=1000,
+    context=[],
+)
+def rebuild_toolchains_action(parameters, graph_config, input, task_group_id, task_id):
+    decision_task_id, full_task_graph, label_to_taskid = fetch_graph_and_labels(
+        parameters, graph_config, task_group_id=task_group_id
+    )
+    toolchains = [label for label, task in full_task_graph.tasks.items() if task.kind == "toolchain"]
```
Comment (bhearsum) [inline taskcluster/translations_taskgraph/actions/rebuild_toolchains.py:21]: I think we should rename this action and include docker images and fetches here as well, as they'll cause all the same problems as toolchains do.

**Review by bhearsum — CHANGES_REQUESTED** [review]
Great idea! We should include a couple of more things here though.

File: taskcluster/translations_taskgraph/actions/rebuild_toolchains.py (hunk 1 above)
Comment (gabrielBusta) [reply taskcluster/translations_taskgraph/actions/rebuild_toolchains.py]: Mm, any ideas on names? We can create those kinds too.

Comment (bhearsum) [reply taskcluster/translations_taskgraph/actions/rebuild_toolchains.py]: How about "Rebuild docker images and toolchains". (I don't think "fetch" needs to be included in the names, seeing as they just feed toolchains.)

**Review by bhearsum — APPROVED** [review]
This looks fine; can you make sure you test it explicitly either here or in staging before this merges, please?

**Review by eu9ene — APPROVED** [review]
Looks good, but I suggest to add a comment why it's needed and when to use it. Ideally even document it in the docs.

**Comment by bhearsum** [comment]
I'll address those comments and drive this home.

**Review by eu9ene — APPROVED** [review]
Thanks for the docs!

**Pull request merged**